Userland/Utilities/tar.cpp

```diff
@@ -223,18 +195,19 @@ int main(int argc, char** argv)
             while (it.has_next()) {
                 auto child_path = it.next_full_path();
                 if (!Core::File::is_directory(child_path)) {
-                    add_file(child_path);
+                    TRY(add_file(child_path));
                 } else {
-                    handle_directory(child_path, handle_directory);
+                    TRY(handle_directory(child_path, handle_directory));
                 }
             }
+            return ErrorOr<void>();
```

Comment:
Please use the implicit default constructor through `{}`

Reply:
Done.